AndroidAnnotations/functional-test-1-5/src/main/java/org/androidannotations/test15/ActivityWithServices.java

```diff
@@ -121,6 +125,9 @@
 	@SystemService
 	DropBoxManager dropBoxManager;
 	
+	// @SystemService
+	// EthernetManager ethernetManager; // hidden API
```

Comment:
no issue. just saw that and wonderd what it means. i'm ok with it if it works. have you tested to use that to inject a field? i once played with an hidden api and had to use reflection to use it as the android.jar that is used for compilation did not contain the method.